patches/chrome-browser-performance_manager-chrome_browser_main_extra_parts_performance_manager.cc.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc b/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
+index 50be376ee5b6043847cfd3b55aaabadc74e15da6..80270faae7278c48b3b70432647e7cb68469f86e 100644
+--- a/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
++++ b/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
+@@ -201,10 +201,13 @@ void ChromeBrowserMainExtraPartsPerformanceManager::CreatePoliciesAndDecorators(
+       std::make_unique<performance_manager::policies::MemorySaverModePolicy>());
+ #endif  // !BUILDFLAG(IS_ANDROID)
+ 
++
++#if 0
+   graph->PassToGraph(
+       std::make_unique<performance_manager::metrics::MemoryPressureMetrics>());
+   graph->PassToGraph(
+       std::make_unique<performance_manager::metrics::PageResourceMonitor>());
++#endif
```

Comment:
tbh the option of redefining these two with an empty implementation doesn't look that bad now.